internal/classification-rubric.md

```diff
@@ -0,0 +1,20 @@
+# Classification Rubric
+
+This is an initial set of guidelines to use
+when deciding on the classification of
+[fields for API/SDK resources](./adding-a-new-field-readme.md#add-new-fields-to-the-apisdk-resource-protobufs).
+
+Fields can be classified as `secret`, `sensitive`, or `public`.
+If a field is not classified, it is treated as `secret`.
+See the [filters/encrpyt][encrpt] package for more details.
+
+* **secret**: This classification should be used for any field that contains
+    information like tokens, keys, passwords.
+* **sensitive**: This classification should be used for any field that contains
+    information like [personally identifiable information (PII)][PII], such as
+    names of people, email addresses, etc.
+* **public**: This classification should be use for any field that is not
+    sensitive or secret.
+
```

Comment:
Can we add a section on who to ask if the classification is not clear?

Reply:
not sure what to put for this actually

Reply:
> Tag @jimlambrt and @jefferai in your PR if you are unsure about how to classify a field.